### ISSUE TITLE
support for passing connection string instead of passing the whole object

### ISSUE THREAD

**Issue by adnan-kamili** [typeorm/typeorm#656]
Instead of passing the connectionOptions object can you allow for passing connection string, that is very common in ORMs, so instead of:

```javascript
const connection = await createConnection({
            type: "mysql",
            host: "localhost",
            port: 3306,
            username: "root",
            password: "root",
            database: "test_db2",
            entities: [
                __dirname + "/models/*.js"
            ],
            autoSchemaSync: true
        });
```

I can simply do:
```javascript
const connection = await createConnection({
            connectionUri: config.connectionUri, //"mysql://root:root@localhost:3306/test_db"
            entities: [
                __dirname + "/models/*.js"
            ],
            autoSchemaSync: true
        });
```

**Comment by yonilerner** [comment]
I could be wrong, but Im pretty sure on the current version published to NPM (`0.1.0-alpha.31`), you can do 
```javascript
createConnection({
  url: 'someconnectionuri',
  ...
})
```

**Comment by adnan-kamili** [comment]
Yes, following worked for me:

const connection = await createConnection({
            type: "mysql",
            url: dbConfig.connectionUri,
            entities: [
                __dirname + "/models/*.js"
            ],
            autoSchemaSync: true
        });
